packages/vuetify/src/components/VBtn/__tests__/VBtn.spec.cy.tsx

```diff
@@ -125,6 +125,29 @@ describe('VBtn', () => {
     })
   })
 
+  describe('loading', () => {
+    it('when using the loader slot, do not show the progress indicator', () => {
+      cy.mount(() => (
+        <VBtn loading v-slots={ { loader: () => <span>{ loadingText }</span> } } />
+      ))
+        .get('button')
+        .should('contain.text', loadingText)
+        .get('[role="progressbar"]')
+        .should('not.exist')
+    })
+
+    // custom loaders are not yet implemented
+    it('when loading is true, show the progress indicator', () => {
+      cy.mount(<VBtn loading>{ loadingText }</VBtn>)
+        .get('button')
+        .should('contain.text', loadingText)
+        .get('.v-btn__content')
+        .should('not.be.visible')
+        .get('[role="progressbar"]')
+        .should('be.visible')
+    })
+  })
```

Comment:
Kinda pointless really, just the visual test is fine

Reply:
What you mean ? I just get existing disabled test and add some fixes.

Reply:
I mean these can be deleted, it's already covered on line 29

Reply:
In which case it cover "default progress does not exist with slot but slot is exist", "btn text does not visible when in loading state" and "loading progress bar should be visible" ?

Reply:
By the way - `btn__content` should be around ALL content including pre/post icons/slots